tests/features/collection/og.feature

```diff
@@ -83,27 +82,3 @@ Feature: Organic Groups integration
     Then I should see the success message "You are no longer a member of Folk Dance and Song Society."
     And I should see the "Join this collection" button
     And the "Folk Dance and Song Society" collection should have 0 members
-
-  @terms
-  Scenario: Edit a Collection
-    Given the following owner:
-      | name                 | type                    |
```

Comment:
I moved this test to `tests/features/collection/collection.edit.feature` so that it is in the same place as all the other edit tests. I don't know how this found its way into this completely unrelated test.